cmd/nerdctl/container_run_network_linux_test.go

```diff
@@ -575,3 +575,73 @@ func TestRunContainerWithMACAddress(t *testing.T) {
 		}
 	}
 }
+
+func TestRunContainerWithStaticIP6(t *testing.T) {
```

Comment:
IPv6 tests should require a new test CLI flag like `-test.ipv6=true`
https://github.com/containerd/nerdctl/blob/1642bef99f0931b79fe75fd54c1566e9e061b178/pkg/testutil/testutil.go#L507

Reply:
Make sense